conda/recipes/libcugraph/meta.yaml

```diff
@@ -38,9 +38,9 @@ requirements:
   host:
     - doxygen {{ doxygen_version }}
     - cudatoolkit {{ cuda_version }}.*
+    - libraft-headers {{ minor_version }}.*
+    - libraft-distance {{ minor_version }}.*
```

Comment:
I think this might also need to be added to pylibcugraph and cugraph

Reply:
Done in latest push